dali/python/nvidia/dali/ops.py

```diff
@@ -1184,26 +1203,27 @@ def _to_type_desc(input):
     if type(input) == int:
         return "int32"
     if type(input) == float:
-        return "float32" # TODO(klecki): current DALI limitation
+        return "float32"  # TODO(klecki): current DALI limitation
     if isinstance(input, _ScalarConstant):
         dtype_to_desc = {
-            DALIDataType.BOOL:    "bool",
-            DALIDataType.INT8:    "int8",
-            DALIDataType.INT16:   "int16",
-            DALIDataType.INT32:   "int32",
-            DALIDataType.INT64:   "int64",
-            DALIDataType.UINT8:   "uint8",
-            DALIDataType.UINT16:  "uint16",
-            DALIDataType.UINT32:  "uint32",
-            DALIDataType.UINT64:  "uint64",
+            DALIDataType.BOOL: "bool",
+            DALIDataType.INT8: "int8",
+            DALIDataType.INT16: "int16",
+            DALIDataType.INT32: "int32",
+            DALIDataType.INT64: "int64",
+            DALIDataType.UINT8: "uint8",
+            DALIDataType.UINT16: "uint16",
+            DALIDataType.UINT32: "uint32",
+            DALIDataType.UINT64: "uint64",
             DALIDataType.FLOAT16: "float16",
-            DALIDataType.FLOAT:   "float32",
+            DALIDataType.FLOAT: "float32",
             DALIDataType.FLOAT64: "float64",
         }
         return dtype_to_desc[input.dtype]
-    raise TypeError(("Constant argument to arithmetic operation not supported. Got {}, expected "
-            "a constant value of type 'bool', 'int', 'float' or 'nvidia.dali.types.Constant'.")
-            .format(str(type(input))))
+    raise TypeError(
+        ("Constant argument to arithmetic operation not supported. Got {}, expected "
+         "a constant value of type 'bool', 'int', 'float' or 'nvidia.dali.types.Constant'.").format(
+             str(type(input))))
```

Comment:
Drop extra parenthesis and use `f".."` syntax?

Reply:
Done